apps/user-service/src/test/resources/application-test.yml

```
@@ -1,14 +1,14 @@
+    password: 12345
+  cloud:
+    discovery:
+      enabled: false
   jpa:
-    database-platform: org.hibernate.dialect.H2Dialect
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
     hibernate:
-      ddl-auto: update
\ No newline at end of file
+      ddl-auto: create-drop
+    show-sql: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)